tf/cloud-function-v2/cloud-function.tf

```diff
@@ -0,0 +1,103 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+resource "google_cloudfunctions2_function" "function" {
+  name     = "e2etest-${terraform.workspace}"
+  location = "us-central1"
+
+  timeouts {
+    create = "5m"
+  }
+
+  build_config {
+    runtime = var.runtime
+    entry_point = var.entrypoint
+    source {
+      storage_source {
+        bucket = google_storage_bucket.bucket.name
+        object = google_storage_bucket_object.object.name
+      }
+    }
+  }
+
+  service_config {
+    max_instance_count = 1
+    available_memory = "256M"
+    timeout_seconds = 540
```

Comment:
Is this the timeout for the function to run and does it need to be so long?

Reply:
No, it does not. This specific value is the max possible. I updated it during testing, but forgot to switch it back. Thinking of changing it to `120` seconds.

It does not take 2 minutes, but wanted to keep some buffer to avoid flakiness due to internet connectivity/speed issues. 
Thoughts ?